renovate.json

```diff
@@ -0,0 +1,48 @@
+{
+  "$schema":"https://docs.renovatebot.com/renovate-schema.json",
+  "extends":[
+     "config:base"
+  ],
+  "timezone":"America/Los_Angeles",
+  "schedule":[
+     "after 3am on the 1st and 3rd day instance on Monday"
+  ],
+  "enabledManagers":[
+    "npm"
+ ],
+ "vulnerabilityAlerts": {
+   "labels": ["security"],
+   "schedule": ["at any time"]
+ },
+  "rangeStrategy":"replace",
+  "npm":{
```

Comment:
Can't find where I got this syntax from at the moment 🤔 It definitely works b/c it excludes the recent node types update from 2 days ago. Let me remove it and verify that the node types update pops in just to double check.

Reply:
So I think for the presets listed in the docs, you can put them at the top level *or* to apply for a particular package. So in our case since we are limiting to npm at the top level,  I think we can apply the stabilityDays to everything (if we wanted to also target infra dependencies and have a different configuration, then we would target by package. 

So I think we can simplify what I had even further just to be the stabilityDays key.